list_enumerable.rb

```diff
@@ -0,0 +1,25 @@
+class MyList
+  include MyEnumerable
+
```

Comment:
- [ ] <b>REQUIRED:</b> If we run this file as it is, we will get an error: 
![Screen Shot 2022-04-14 at 23 33 58](https://user-images.githubusercontent.com/45217364/163480696-6e47a5c1-b092-4486-874e-44d5288a342d.png)

That is because we don't have access to the MyEnumerable module as it is a different file. So to have access to it, we simply add `require_relative 'module_enumerable.rb'` on top of the file to import everything from that file.

Reply:
@rukundoeric thank you very much for the time taken and this tip. You could also import the files in Interactive Ruby (IRB) with these 2 commands:
- first type `require './module_enumerable.rb'`
- then type  `require './list_enumerable.rb'`
Once you do this, everything is available without the need to add `require_relative 'module_enumerable.rb'` on top of the file to import the module, and you could easily run all the tests.
Anyway, the asked change is implemented. `require_relative 'module_enumerable.rb'` on top of the file.